From 33bc1de71f52880842d5410a1c08ba5778afb016 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Klaus=20K=C3=A4mpf?= <kkaempf@suse.de>
Date: Tue, 28 Nov 2017 15:32:37 +0100
Subject: [PATCH 20/29] link certificate if missing

On updates from older versions that did not use clist.pem, we
need to properly link the file here or service will not start
---
 genSslCert.sh | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/genSslCert.sh b/genSslCert.sh
index 9bf6320d5525..17e26256f301 100644
--- a/genSslCert.sh
+++ b/genSslCert.sh
@@ -14,6 +14,13 @@ then
     echo "WARNING: server.pem SSL Certificate file already exists."
     echo "         old file will be kept intact."
     DO_SERVER=no
+
+    # On updates from older versions that did not use clist.pem, we
+    # need to properly link the file here or service will not start
+    if [ ! -e $TARGETDIR/clist.pem ]
+    then
+        ln -s server.pem $TARGETDIR/clist.pem
+    fi
 fi
 
 if [ -f $TARGETDIR/client.pem ]
-- 
2.23.0

